### PR TITLE
Fix issues with report an issue page

### PR DESCRIPTION
**Pull request by atavism**
Resolves https://github.com/getlantern/engineering/issues/1361

- [x] When sending a report, it's not obvious that a description is required. Can we make the description optional?
- [x] Always show "Describe your issue" placeholder text
- [x] Do not show "send_report" label on mouse hover of Send Report button

### CODE REVIEW

File: lib/common/session_model.dart [hunk 1]
```diff
@@ -82,11 +82,17 @@ class SessionModel extends Model {
...
   late ValueNotifier<String?> country;
 
-  // wsMessageProp parses the given json, checks if it represents a pro user message and
-  // returns the value (if any) in the map for the given property.
-  String? wsMessageProp(Map<String, dynamic> json, String field) {
-    if (json["type"] != "pro") return null;
-    return json["message"][field];
+  // listenWebsocket listens for websocket messages from the server. If a message matches the given message type,
+  // the onMessage callback is triggered with the given property value
+  void listenWebsocket<T>(WebsocketImpl? websocket, String messageType, property, void Function(T?) onMessage) {
```
Comment (jigar-f) [inline lib/common/session_model.dart:87]: That's a neat approach, looks good! 👍

File: lib/common/session_model.dart [hunk 2]
```diff
@@ -305,12 +294,16 @@ class SessionModel extends Model {
...
   }
 
-  Future<void> setProxyAll<T>(bool on) async {
-    unawaited(
-      methodChannel.invokeMethod('setProxyAll', <String, dynamic>{
-        'on': on,
-      }),
-    );
+  Future<void> setProxyAll<T>(bool isOn) async {
+    if (isMobile()) {
```
Comment (jigar-f) [inline lib/common/session_model.dart:298]: I think we need to remove Mobile call here, We don't have any method for proxyAll

**Review by jigar-f — APPROVED** [review]
LGTM!

File: lib/common/session_model.dart (hunk 2 above)
Comment (atavism) [reply lib/common/session_model.dart]: Will do!

File: lib/common/session_model.dart (hunk 1 above)
Comment (atavism) [reply lib/common/session_model.dart]: Thanks!

**Comment by atavism** [comment]
Thanks for the review, @jigar-f!

**Pull request merged**